en/developer_guide/sitemap.rst

```diff
@@ -0,0 +1,83 @@
+Using sitemap.xml
+=================
+
+As Open Orchestra is a multisite platform, you will have to manage several sitemap.xml files for a
+single installation. To simplify the maintenance of their content, information contained in the files
+can be directly edited in the Back Office. But contribution is only the first step. To be available,
+these information require to be dumped in files and to be correctly served to the client.
+
+Creation of ``sitemap.xml`` files with Open Orchestra is achieved in 3 steps:
+
+- `files contribution`_
+- `files generation`_
+- `files routing`_
+
+.. _files contribution:
+
+Files contribution
+------------------
+Contributing the content of a ``sitemap.xml`` file is done in the Backoffice, and two locations
+are involved: one for the default values, and one for the specific values.
+
+The first place to go is website edition. You can fill in the edit form default values for all pages
```

Comment:
In the website edition form, you could set up the default values (see website creation doc for further information).